src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/Helpers.kt

```diff
@@ -159,4 +161,22 @@ private val prettyTime = PrettyTime()
 fun getFriendlyRelativeDateTime(date: Date) : String
 {
     return prettyTime.format(date)
+}
+
+fun jsonToStringMap(json: String): Map<String, String>
+{
+    val element = JsonParser().parse(json)
```

Comment:
You don't need to parse the JSON here - for efficiency you can pass the text directly to `fromJson`